source/core.py

```diff
@@ -221,6 +219,8 @@ def getWxLang(lang):
 	# Therefore treat these situations like wx not knowing the language at all.
 	if wxLang and not locale.IsAvailable(wxLang.Language):

```

Comment:
I obviously misread this the first time. I missed the check for `None` at the start of this condition.